hera_sim/tests/test_vis.py

```diff
@@ -5,7 +5,11 @@
 
 from hera_sim import visibilities as vis, io
 
-SIMULATORS = (vis.HealVis, vis.VisCPU)
+try:
+    SIMULATORS = (vis.HealVis, vis.VisCPU)
+except AttributeError: # If healvis is not imported
```

Comment:
I don't think this is the type of error we would want to let pass silently, especially if we're testing the functionality of these simulators. At the very least, I would recommend adding a warning if the `except` clause is executed. Something along the lines of this:
```
except AttributeError:
    warnings.warn("healvis failed to import in the package constructor.")
    ...
```

Reply:
Should I include this if I've added the warning in the __init__ file that it did not import?

Reply:
I suppose it's not necessary, but it couldn't hurt to throw it in there. That way when `pytest` gives it report, we'll see that there was a warning specifically in this test.

Reply:
Done.